modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java

```diff
@@ -50,6 +51,10 @@ protected OMElement serializeEndpoint(Endpoint endpoint) {
 
         serializeCommonAttributes(endpoint,endpointElement);
 
+        if (failoverEndpoint.isBuildMessageAtt()) {
+            failoverElement.addAttribute(XMLConfigConstants.BUILD_MESSAGE, Boolean.toString(failoverEndpoint.isBuildMessageAtt()), null);
```

Comment:
Is line 55 in 120 limit?

Reply:
Fixed in https://github.com/wso2/wso2-synapse/pull/892